tests/sensors/test_external_task_sensor.py

```diff
@@ -1478,7 +1478,7 @@ def dummy_task(x: int):
             mode="reschedule",
         )
 
-        body = dummy_task.expand(x=[i for i in range(5)])
+        body = dummy_task.expand(x=list(range(5)))
```

Comment:
Actually I don’t think even the `list` is needed, `expand` should be able to accept `range` since it implements the sequence protocol.